src/pint/data/runtime/observatories.json

```diff
@@ -1520,8 +1530,6 @@
         "origin": "An Imaging Atmospheric Cherenkov Telescope.\nThese coordinates are provided (from geodetic coordinates : lat = 23°16'18''S, lon = 16°30'00''E at 1800m asl) by the Maxime Regeard and Arache Djannati-Atai on behalf of the H.E.S.S. Collabotation, 2023 January 18."
     },
     "hawc": {
-        "include_bipm": false,
-        "include_gps": true,
```

Comment:
Isn't it better to set apply_gps2utc as false explicitly?

I am assuming that's the default behaviour.

Reply:
The default for `include_bipm` is true (I didn't change that).
However, I agree that having it set explicitly for each observatory is good.
I've done that for all the observatories where it was obvious (e.g. the clock file was XXX2gps.clk).
The correct choice for other observatories will need to come from people with specific knowledge of how that observatory does time stamps.